x-pack/test/case_api_integration/basic/tests/configure/get_connectors.ts

```diff
@@ -61,23 +65,33 @@ export default ({ getService }: FtrProviderContext): void => {
         .send(getJiraConnector())
         .expect(200);
 
+      const { body: connectorFour } = await supertest
+        .post('/api/actions/action')
+        .set('kbn-xsrf', 'true')
+        .send(getResilientConnector())
+        .expect(200);
+
       actionsRemover.add('default', connectorOne.id, 'action', 'actions');
       actionsRemover.add('default', connectorTwo.id, 'action', 'actions');
       actionsRemover.add('default', connectorThree.id, 'action', 'actions');
+      actionsRemover.add('default', connectorFour.id, 'action', 'actions');
 
       const { body: connectors } = await supertest
         .get(`${CASE_CONFIGURE_CONNECTORS_URL}/_find`)
         .set('kbn-xsrf', 'true')
         .send()
         .expect(200);
 
-      expect(connectors.length).to.equal(2);
+      expect(connectors.length).to.equal(3);
```

Comment:
I feel like this test can be really annoying each time someone adds a new connector, I think the last three expect should be sufficient.

Reply:
I know you did not write the test, but we should have a different test for each connector. It will be cleaner.